test/Microsoft.Extensions.DependencyModel.Tests/PackageResolverTest.cs

```diff
@@ -24,8 +24,8 @@ public void ShouldUseEnvironmentVariableToGetDefaultLocation()
                 .AddVariable("NUGET_PACKAGES", PackagesPath)
                 .Build();
 
-            var result = PackageCompilationAssemblyResolver.GetDefaultPackageDirectory(Platform.Unknown, environment);
-            result.Should().Be(PackagesPath);
+            var result = PackageCompilationAssemblyResolver.GetDefaultProbeDirectories(Platform.Unknown, environment);
```

Comment:
> this is not very common scenario.

It is becoming more and more common.  For example, in the latest 2.0 SDK, the "first run"/"offline" experience uses a fallback folder instead of populating the NuGet cache.  So by default, everyone will be using fallback folders + NuGet cache.